ui/components/src/UnitSelector/UnitSelector.tsx

```diff
@@ -105,17 +104,17 @@ export function UnitSelector({ value, onChange }: UnitSelectorProps) {
         }
       />
       <OptionsEditorControl
-        label="Decimal"
+        label="Decimals"
```

Comment:
Changed label for `decimal_places` from `Decimal` --> `Decimals`

Reply:
Should the label actually be 'Max Decimals'?

Reply:
I'll check with Mohit

Reply:
Sounds like no -- 

After this change goes in, we can go back to: if the user sets decimal_places, then we always show that exact number of decimal places. 

(Before, that was causing problems because we didn't have the default option)